proto/osmosis/pool-incentives/v1beta1/incentives.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+package osmosis.poolincentives.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/pool-incentives/types";
+
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+
+  // minted_denom is the denomination of the coin expected to be minted by the minting module.
+  // Pool-incentives module doesn’t actually mint the coin itself, but rather manages the distribution of coins that matches the defined minted_denom.
+  string minted_denom = 1 [(gogoproto.moretags) = "yaml:\"minted_denom\""];
```

Comment:
This minted_denom has possibility to not be bond denom?

Reply:
No, it should be set to the same in genesis.